### PR TITLE
[5.6] Feature/refactor Str::languageSpecificCharsArray()

### PR DESCRIPTION
**Pull request by SergeyPodgornyy**
## Description

Improve readability of `Support\Str::languageSpecificCharsArray()`

## Motivation and context

Associative array could be easier read and support as to different arrays, because you can always see which value refers to which value

## How has this been tested?

All tests passed successfully

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [x] I have run tests to cover my changes.

### CODE REVIEW

File: src/Illuminate/Support/Str.php [hunk 1]
```diff
@@ -56,8 +56,8 @@ public static function ascii($value, $language = 'en')
     {
         $languageSpecific = static::languageSpecificCharsArray($language);
 
-        if (! is_null($languageSpecific)) {
```
Comment (SergeyPodgornyy) [inline src/Illuminate/Support/Str.php]: https://github.com/kalessil/phpinspectionsea/blob/master/docs/probable-bugs.md#null-pointer-exceptions-prevention

> is_object(), is_null() and similar is_*() functions calls are not analyzed, instead we recommend to rely on null identity and instanceof operators;

All 'is_null(...)' calls can be safely replaced with 'null === ...' constructs (or 'null !== ...' if the original construct was negated).

Comment (ntzm) [reply src/Illuminate/Support/Str.php]: The rest of the framework uses `is_null`

**Comment by sisve** [comment]
I understand what you're trying to do, but this part of the code is taken from the Stringy library. Your changes will be lost next time we update this method.

https://github.com/danielstjules/Stringy/blob/df24ab62d2d8213bbbe88cc36fc35a4503b4bd7e/src/Stringy.php#L1869-L1878

File: src/Illuminate/Support/Str.php (hunk 1 above)
Comment (SergeyPodgornyy) [reply src/Illuminate/Support/Str.php]: @ntzm yes, but it will be more clear and correct to use `null !== ...`. Should I revert this changes?

Comment (ntzm) [reply src/Illuminate/Support/Str.php]: It should stay consistent throughout the framework

**Comment by SergeyPodgornyy** [comment]
@sisve hm, good notice :+1: Should I contribute then to `danielstjules/stringy`?

File: src/Illuminate/Support/Str.php (hunk 1 above)
Comment (SergeyPodgornyy) [reply src/Illuminate/Support/Str.php]: @ntzm Rolled back

**Comment by SergeyPodgornyy** [comment]
Pull Request to original repository in progress https://github.com/danielstjules/Stringy/pull/179

**Pull request closed without merging**